src/test/java/org/jabref/logic/citationkeypattern/BracketedPatternTest.java

```diff
@@ -76,22 +77,65 @@ void allAuthors(String expected, AuthorList list) {
 
     static Stream<Arguments> authorsAlpha() {
         return Stream.of(
-                Arguments.of("A+", "Alexander Artemenko and others"),
-                Arguments.of("A+", "Aachen and others"),
-                Arguments.of("AB+", "Aachen and Berlin and others"),
-                Arguments.of("ABC+", "Aachen and Berlin and Chemnitz and others"),
+                Arguments.of("Ar", "Alexander Artemenko and others"),
+                Arguments.of("Aa", "Aachen and others"),
+                Arguments.of("Aa", "Aachen and Berlin and others"),
+                Arguments.of("Aa", "Aachen and Berlin and Chemnitz and others"),
+                Arguments.of("AB", "Aachen and Berlin"),
+                Arguments.of("ABC", "Aachen and Berlin and Chemnitz"),
                 Arguments.of("ABCD", "Aachen and Berlin and Chemnitz and Düsseldorf"),
-                Arguments.of("ABC+", "Aachen and Berlin and Chemnitz and Düsseldorf and others"),
-                Arguments.of("ABC+", "Aachen and Berlin and Chemnitz and Düsseldorf and Essen"),
-                Arguments.of("ABC+", "Aachen and Berlin and Chemnitz and Düsseldorf and Essen and others"));
+                Arguments.of("Aa", "Aachen and Berlin and Chemnitz and Düsseldorf and others"),
+                Arguments.of("ABCD", "Aachen and Berlin and Chemnitz and Düsseldorf and Essen"),
```

Comment:
As per our discussion, we would do that if there is "and others".

Reply:
This has been established ever since we started talking about DIN. I can do only the first author's two letters of last name in case of exceeding authors but note that everywhere I have looked, in case of exceeding authors (and not "and others"), the abbreviation of first four authors are taken.
You can try citing 10.3945/ajcn.111.023457 using citationmachine as well: https://www.citationmachine.net/bibliographies/a446c586-d900-4583-a249-72d7b9bc0b52
![image](https://github.com/user-attachments/assets/b8b803ae-b7fa-40a4-8749-633c9d451983)

Reply:
Besides the LNI template: https://github.com/JabRef/jabref/pull/11614#issuecomment-2284786529

We need to setup a minimal latex+bibtex document with authorsalpha with the test cases and see what that returns.